scripts/abi_artifact_generator/config/architectures.yml

```diff
@@ -1,98 +1,126 @@
 ---
 "x86_64":
   register_type: uint64_t
+  intel_register_type: qword
   register_size: 8
   register_list: [rax, rbx, rcx, rdx, rsi, rdi, r8, r9, r10, r11, r12, r13, r14, r15]
+  styles:
+    gcc: "# "
+    msvc: ";; "
   templates:
     call_a_function: |
       call {{ function_name }}
     return_from_function: |
       ret
-    return_from_function_with_cleanup: |
-      ret $8
     fill_a_register: |
-      mov {{ location }} + {{ offset }}, %{{ register }}
+      mov {{ register }}, qword ptr [{{ location }} + {{ offset }}]
```

Comment:
INTEL SYNTAX \o/

Reply:
I still prefer AT&T (I guess because my first (and favorite) assembly is ARM. But MASM really doesn't like it :enterprise: